src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/help-coberturaChkBx.html

```diff
@@ -0,0 +1,5 @@
+<div>
+<br>
+Generate a Cobertura code coverage report, named <b> cobertura.xml, </b> to be stored in the<b> matlabTestArtifacts</b> folder of the current Jenkins workspace.
```

Comment:
Remove extraneous spaces around `cobertura.xml,`

`named <b> cobertura.xml, </b> to` -> `named <b>cobertura.xml</b>, to`

Reply:
Done.